src/Mailer/Email.php

```diff
@@ -1484,6 +1485,17 @@ protected function _encode($text)
     }
 
     /**
+     * Decode the specified string
+     *
+     * @param string $text String to decode
+     * @return string Decoded string
+     */
+    protected function _decode($text)
+    {
+        return mb_decode_mimeheader($text);
```

Comment:
Since he wrote them we might as well keep them maybe.
After all, there could be a typo in the way the protected method is being called from getOriginalSubject() etc.